zgw_consumers/constants.py

```diff
@@ -14,3 +14,9 @@ class APITypes(DjangoChoices):
     kic = ChoiceItem("kic", _("KIC (Klantinteracties)"))
 
     orc = ChoiceItem("orc", _("ORC (Overige)"))
+
+
+class AuthTypes(DjangoChoices):
+    no_auth = ChoiceItem("no_auth", _("No authorization"))
+    token = ChoiceItem("token", _("Token"))
+    jwt = ChoiceItem("jwt", _("JWT"))
```

Comment:
I'd name this `ZGW Client ID + secret` rather. JWT has many different forms and is more of a mechanism.

Reply:
renamed to `zgw`